src/Microsoft.Health.Fhir.Core/Messages/Export/CreateExportRequest.cs

```diff
@@ -0,0 +1,31 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using EnsureThat;
+using MediatR;
+using Microsoft.Health.Fhir.Core.Features.Operations;
+using Newtonsoft.Json;
+
+namespace Microsoft.Health.Fhir.Core.Messages.Export
+{
+    public class CreateExportRequest : IRequest<CreateExportResponse>
+    {
+        public CreateExportRequest(Uri requestUri)
+        {
+            EnsureArg.IsNotNull(requestUri, nameof(requestUri));
+
+            RequestUri = requestUri;
+        }
+
+        [JsonConstructor]
+        protected CreateExportRequest()
+        {
+        }
+
+        [JsonProperty(JobRecordProperties.RequestUri)]
+        public Uri RequestUri { get; private set; }
```

Comment:
nit: this can be simplified as `public Uri RequestUri { get; }`.

Reply:
I am storing the whole `CreateExportRequest` object as part of `ExportJobRecord` and hence need the private setter so that deserialization works. But now that I think about it more, I guess I can just store the `RequestUri` instead of the entire request.